client/src/hooks/useCreateUser.js

```diff
@@ -48,6 +51,16 @@ export const useCreateUser = (
     needsSave = false
   }
 
+  React.useEffect(() => {
```

Comment:
Could you define something like this here and remove the guard from the context?
```javascript
const handleChangeStart = (url) => {
  if (!url.includes('orcid.org') && !url.includes('create-account')) {
    cleanup()
  }
}
```